lib/login.js

```diff
@@ -261,11 +263,18 @@ module.exports = {
         }
 
         const profile = await this._loadProfileAsync(profileName);
-        const loginUrl = await this._createLoginUrlAsync(profile.azure_app_id_uri, profile.azure_tenant_id);
+        let assertionConsumerServiceURL = AWS_SAML_ENDPOINT;
+        if (profile.hasOwnProperty('region') && (profile.region.indexOf('us-gov-west') !== -1 || profile.region.indexOf('us-gov-east') !== -1)) {
```

Comment:
Why not just do `['us-gov-west', 'us-gov-east'].includes(profile.region)` here?

Reply:
Currently the only GovCloud regions are `us-gov-east-1` and `us-gov-west-1`, but I wanted to account for future regions such as `us-gov-east-2`. What I have is still not the clearest way to do that, but I think this is better:

`profile.region.startsWith('us-gov')`

Reply:
That looks better, thanks!